services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/ErrorOperationsBolt.java

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.nbworker.bolts;
+
+import org.openkilda.messaging.Destination;
+import org.openkilda.messaging.Message;
+import org.openkilda.messaging.Utils;
+import org.openkilda.messaging.error.ErrorData;
+import org.openkilda.messaging.error.ErrorMessage;
+import org.openkilda.wfm.AbstractBolt;
+import org.openkilda.wfm.error.AbstractException;
+import org.openkilda.wfm.error.JsonEncodeException;
+import org.openkilda.wfm.topology.AbstractTopology;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.storm.topology.OutputFieldsDeclarer;
+import org.apache.storm.tuple.Tuple;
+import org.apache.storm.tuple.Values;
+
+public class ErrorOperationsBolt extends AbstractBolt {
+    @Override
+    protected void handleInput(Tuple input) throws AbstractException {
+        ErrorData data = (ErrorData) input.getValueByField("request");
```

Comment:
The `pullValue` is useful. I'll change it here. 
The bolt just reroute messages to NB. This behavior can be used as default for general error. In case when additional work must be done, we can create new Bolt for handle an error and make `if` section in `RouterBolt` for it.
You are right about constants, I'll create class for them.